backend/climateconnect_api/views/user_views.py

```diff
@@ -140,7 +141,10 @@ def get_serializer_class(self):
         return UserProfileStubSerializer
 
     def get_queryset(self):
-        user_profiles = UserProfile.objects.filter(is_profile_verified=True)
+        user_profiles = UserProfile.objects\
+            .filter(is_profile_verified=True)\
+            .annotate(is_image_null=Count("image"))\
```

Comment:
It seems like an empty image field defaults to `""`(empty string) rather than `null`.
In order for this value to be true when the user doesn't have an image, you'll have to use an additional filter:
`.annotate(is_image_null=Count('image', filter=Q(image="")))`.

Reply:
Yeah, you are right, thank you. I have update PR.